packages/page-accounts/src/Accounts/index.tsx

```diff
@@ -112,13 +66,43 @@ function Overview ({ className = '', onStatusChange }: Props): React.ReactElemen
   const [{ balanceTotal }, setBalances] = useState<Balances>({ accounts: {} });
   const [sortedAccounts, setSortedAccounts] = useState<SortedAccount[]>([]);
   const [filterOn, setFilter] = useState<string>('');
+  const [sortedAccountsWithDelegation, setSortedAccountsWithDelegation] = useState<SortedAccount[]>([]);
+  const sortedAddresses = sortedAccounts.map((a) => a.account.address);
```

Comment:
Since delegations aren't related to account sorting, I choose to keep it outside of `sortAccount`. To get the benefit of a .multi call for delegation, I needed to retrieve the sorted account addresses here.
lmk if there's a better way to do.

Reply:
The issue is just that the variable will change each render, so we are relying on the useCall de-dupe to not fire. 

So would probably change the `useState<SortedAccount[]>([]);` to 

```js
interface Sorted {
  sortedAccounts: SortedAccount[];
  sortedAddresses: string[];
}

..

const [{ sortedAccounts, sortedAddresses }, setSorted] = useState<Sorted>({ sortedAccounts: [], sortedAddresses[] });

...

useEffect((): void => {
  const sortedAccounts = sortAccounts(allAccounts, favorites);
  const sortedAddresses = sortedAccounts.map((a) => a.account.address);

  setSorted({ sortedAccounts, sortedAddresses });
}, [allAccounts, favorites]);

...
```